src/test/scala-2.11/com/linagora/gatling/imap/scenario/ImapExpungeScenario.scala

```diff
@@ -0,0 +1,53 @@
+package com.linagora.gatling.imap.scenario
+
+import java.util.Calendar
+
+import com.linagora.gatling.imap.PreDef._
+import io.gatling.commons.validation.Validation
+import io.gatling.core.Predef._
+import io.gatling.core.scenario.Simulation
+import io.gatling.core.session.Expression
+
+import scala.collection.immutable.Seq
+import scala.concurrent.duration._
+import scala.util.Random
+
+class ImapExpungeScenario extends Simulation {
+  val numOfMailInInbox = 1000
+  val percentageOfMailToExpunge = 20.0
+
+  val userFeeder = csv("users.csv").circular
+  val userCount: Int = userFeeder.records.length
+  def getRandomDeleted: Boolean = Random.nextFloat() < (percentageOfMailToExpunge/ 100.0)
+
+  def flagsWithRandomDeletion: Expression[Session] = (session: Session) => {
+    session.set("flags",
+      if (getRandomDeleted)
```

Comment:
getRandomDeleted() (with parenthesis) make it more obvious that's a fonction call, no?